manifests/vhost.pp

```diff
@@ -2301,6 +2328,11 @@
         include apache::mod::auth_gssapi
       }
 
+      if 'authz_core' in $directory {
+        include apache::mod::authn_file
+        include apache::mod::authz_user
+      }
+
```

Comment:
I wonder if this is needed: I think we can have setups that do not rely on these modules (and having them included unconditionally might be unexpected).

I am not sure we can include classes from a function, but if this is possible that would be the perfect place to make sure required modules are included (e.g. using `Require ldap-group` would include `mod_authz_core`).

If not, this code is probably fine, and maybe other reviewers will propose something better than my "I wonder" :grin:.

Reply:
I took example of `gssapi` and other modules where we are explicitly including different modules baed on type of directory.
The `authz_core` is included in default mod list but not the authn_file one so better I thought to add both mod.

Reply:
I think I am getting now, we don't need it as we already included as default mod. Removed